doc/Makefile

```diff
@@ -79,7 +80,16 @@ kconfig: scripts/genrest/genrest.py
 prep: doxy content kconfig
 
 html: content kconfig
-	$(Q)$(SPHINXBUILD) -t $(DOC_TAG) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	$(Q)$(SPHINXBUILD) -t $(DOC_TAG) -b html $(ALLSPHINXOPTS) \
```

Comment:
I am not sure I like all of this magic in the makefile, you probably want to do all of this in a clean script and just call it here.

Reply:
done